src/main/webapp/courseLoader.js

```diff
@@ -1,127 +1,200 @@
-    const tempHoursList = dataObject.hoursList;
-    const hoursList = [["hours"], /* dummyHourRating */ [3], /* dummyHourRating */ [8]].concat(tempHoursList);
-    const hourData = new google.visualization.arrayToDataTable(hoursList)
-    const hourOptions = {
-        title: "Hours Spent per Week",
-        legend: {position: "none"},
-        vAxis: {title: "# Students"},
-        hAxis: {title: "Hours"},
-        histogram: {
-          hideBucketItems: true
-        }
-      };
-    const hourChart = new google.visualization.Histogram(document.getElementById("hours-chart"));
-    hourChart.draw(hourData, hourOptions);
-
-
-    const tempDiffList = dataObject.hoursList;
-    const diffList = [["difficulty"], /* dummyDifficultyRating */ [1], /* dummyDifficultyRating */ [4]].concat(tempDiffList);
-    const diffData = new google.visualization.arrayToDataTable(diffList)
-    const diffOptions = {
-        title: "Difficulty of Class",
-        legend: {position: "none"},
-        vAxis: {title: "# Students"},
-        hAxis: {title: "Difficulty"},
-        histogram: {
-          hideBucketItems: true
-        }
-      };
-    const diffChart = new google.visualization.Histogram(document.getElementById("diff-chart"));
-    diffChart.draw(diffData, diffOptions);
-
-
-    const tempTermPerceptionList = dataObject.termPerceptionList;
-    const termPerceptionList = [["Term Perception"], /* dummyPerceptionRating */ [11], /* dummyPerceptionRating */ [5]].concat(tempTermPerceptionList);
-    const termPerceptionData = new google.visualization.arrayToDataTable(termPerceptionList);
-    const termPerceptionOptions = {
-        title: "Perception of Term Reviews",
-        legend: {position: "none"},
-        vAxis: {title: "Perception"},
-        hAxis: {title: "Comment Quantity"},
-        histogram: {
-          hideBucketItems: true
-       }
-    };
-    const termPerceptionChart = new google.visualization.Histogram(document.getElementById("term-chart"));
-    termPerceptionChart.draw(termPerceptionData, termPerceptionOptions);
-
-
-    const tempProfPerceptionList = dataObject.termPerceptionList;
-    const profPerceptionList = [["Professor Perception"], /* dummyPerceptionRating */ [2], /* dummyPerceptionRating */ [9]].concat(tempProfPerceptionList);
-    const profPerceptionData = new google.visualization.arrayToDataTable(profPerceptionList);
-    const profPerceptionOptions = {
-        title: "Perception of Professor Reviews",
-        legend: {position: "none"},
-        vAxis: {title: "Perception"},
-        hAxis: {title: "Comment Quantity"},
-        histogram: {
-          hideBucketItems: true
-        }
-      };
-    const profPerceptionChart = new google.visualization.Histogram(document.getElementById("prof-chart"));
-    profPerceptionChart.draw(profPerceptionData, profPerceptionOptions);
+document.addEventListener("DOMContentLoaded", function () {
+  populateData();
+});
+
+function createComment(commentText, flag) {
```

Comment:
flag isn't terribly helpful. what kind of flag is this? What is it determining? Based on the code maybe isCourse would be a good name?